var/spack/repos/builtin/packages/py-aiofiles/package.py

```diff
@@ -15,11 +15,15 @@ class PyAiofiles(PythonPackage):
 
     license("Apache-2.0")
 
-    version("0.7.0", sha256="a1c4fc9b2ff81568c83e21392a82f344ea9d23da906e4f6a52662764545e19d4")
-    version("0.5.0", sha256="98e6bcfd1b50f97db4980e182ddd509b7cc35909e903a8fe50d8849e02d815af")
-
-    depends_on("python@3.6:3", when="@0.7:", type=("build", "run"))
-    depends_on("py-poetry-core@1:", when="@0.7:", type="build")
+    version(
+        "0.7.0",
+        sha256="c67a6823b5f23fcab0a2595a289cec7d8c863ffcb4322fb8cd6b90400aedfdbc",
+        url="https://pypi.org/packages/e7/61/007ac6f27fe1c2dc44d3a62f429a8440de1601428b4d0291eae1a3494d1f/aiofiles-0.7.0-py3-none-any.whl",
+    )
+    version(
+        "0.5.0",
+        sha256="377fdf7815cc611870c59cbd07b68b180841d2a2b79812d8c218be02448c2acb",
+        url="https://pypi.org/packages/f4/2b/078a9771ae4b67e36b0c2a973df845260833a4eb088b81c84b738509b4c4/aiofiles-0.5.0-py3-none-any.whl",
+    )
 
     # Historical dependencies
```

Comment:
Comments can now be removed

Reply:
comments aren't part of `ast.parse` so hard to auto-delete